### PR TITLE
[OSDOCS-9190]:Rewrite and/or remove duplicate information

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-9190
Link to docs preview:

https://69748--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/gcp-ccs
QE review:
- [ ] QE has approved this change.

Additional information:

image

### CODE REVIEW

**Comment by mletalie** [comment]
Reviewers: Please note that I edited the "Customer Cloud Subscriptions on GCP" section to match the "Customer Cloud Subscriptions on AWS" Currently, the docs present duplicate info as shown in attached image: 
image
This change will be more in line with the "Customer Cloud Subscriptions on AWS" overall.

File: osd_planning/gcp-ccs.adoc [hunk 1]
```diff
@@ -7,9 +7,7 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 toc::[]
 
 [role="_abstract"]
-Red Hat recommends the usage of a {GCP} project, managed by the customer, to organize all of your GCP resources. A project consists of a set of users and APIs, as well as billing, authentication, and monitoring settings for those APIs.
-
-It is a best practice for the {product-title} CCS cluster to be hosted in a GCP project within a GCP organization. The Organization resource is the root node of the GCP resource hierarchy and all resources that belong to an organization are grouped under the organization node. An IAM service account with certain roles granted is created and applied to the GCP project. When you make calls to the API, you typically provide service account keys for authentication. Each service account is owned by a specific project, but service accounts can be provided roles to access resources for other projects.
+{product-title} provides a Customer Cloud Subscription (CCS) model that allows Red Hat to deploy and manage clusters into a customer’s existing Google Cloud Platform (GCP) account.
```
Comment (JoeAldinger) [inline osd_planning/gcp-ccs.adoc:10]: Is `into` the correct preposition here? Alternatives could be `on a customer's existing GCP account` or `in a customer's GCP account`.

**Review by JoeAldinger — COMMENTED** [review]
One question about phrasing. Otherwise lgtm.

**Pull request merged**